Client/Frontend/Home/FirefoxHomeViewController.swift

```diff
@@ -761,7 +773,12 @@ extension FirefoxHomeViewController: UICollectionViewDelegateFlowLayout {
         case .pocket:
             return pocketStories.isEmpty ? .zero : getHeaderSize(forSection: section)
         case .topSites:
-            return isTopSitesSectionEnabled ? getHeaderSize(forSection: section) : .zero
+            // Only show a header for top sites if the Firefox Browser is not showing
```

Comment:
Just a bit clearer to me
```suggestion
            // Only show a header for top sites if the Firefox Browser logo is not showing
```